### ISSUE TITLE
Allow plain input in aoc function after generator was defined.

### ISSUE THREAD

**Issue by Smephite** [gobanos/cargo-aoc#60]
It should be possible to implement an generator to return e.g. a `struct`  and one implementation of the solution to use the generated `&struct` but simultaneously another solution to use the default `&str` type.
At the moment ``expected `str`, found struct `struct` `` is thrown.

This is due to the default "pass through" being disabled after implementing a generator.

**Comment by gobanos** [comment]
If I recall correctly you can add `part1` or `part2` to the generator meta in order to use it only for a single part.

**Comment by Smephite** [comment]
Yes, but that does not solve the problem especially since adding a generator for `part1` while a "global" generator is defined will call an exception that a generator for that part is already defined.

On the other hand it would be nice to not need to define a generator returning the input 1:1

**Comment by gobanos** [comment]
I mean mark your "global" generator with `partX` and the other part should immediately get the default generator.

**Comment by Smephite** [comment]
The interesting use behind this would be to benchmark different implementations: one with an generator and one without. 

E.g. One putting the input into an array then solving the problem and one solving the result while reading it thus being more efficient.
